src/silo/database.cpp

```diff
@@ -55,55 +54,37 @@ const PangoLineageAliasLookup& Database::getAliasKey() const {
    {
       BlockTimer const timer(micros);
       partitions.resize(partition_descriptor.partitions.size());
       initializeColumns();
+      initializeSequences();
       for (size_t partition_index = 0; partition_index < partition_descriptor.partitions.size();
            ++partition_index) {
          const auto& part = partition_descriptor.partitions[partition_index];
          partitions[partition_index].chunks = part.chunks;
          for (size_t chunk_index = 0; chunk_index < part.chunks.size(); ++chunk_index) {
-            const std::string name =
-               partition_name_prefix + buildChunkName(partition_index, chunk_index);
-            std::string sequence_filename = name + sequence_file_suffix;
-            const std::filesystem::path metadata_file(name + metadata_file_suffix);
-            if (!InputStreamWrapper(sequence_filename).getInputStream()) {
-               sequence_filename += ".xz";
-               if (!InputStreamWrapper(sequence_filename).getInputStream()) {
-                  SPDLOG_ERROR("Sequence file {} not found", name + sequence_file_suffix);
-                  return;
-               }
-               SPDLOG_DEBUG("Using sequence file: {}", sequence_filename);
-            } else {
-               SPDLOG_DEBUG("Using sequence file: {}", sequence_filename);
-            }
+            std::filesystem::path metadata_file = input_folder;
+            metadata_file += buildChunkString(partition_index, chunk_index) + ".tsv";
             if (!std::filesystem::exists(metadata_file)) {
-               SPDLOG_ERROR("metadata file {} not found", name + metadata_file_suffix);
+               SPDLOG_ERROR("metadata file {} not found", metadata_file.string());
                return;
             }
-            FastaReader sequence_input(sequence_filename);
-            SPDLOG_DEBUG("Using metadata file: {}", name + metadata_file_suffix);
-            const size_t sequence_store_sequence_count =
-               partitions[partition_index].seq_store.fill(sequence_input);
-            const size_t metadata_store_sequence_count =
-               partitions[partition_index].columns.fill(metadata_file, alias_key, database_config);
-            if (sequence_store_sequence_count != metadata_store_sequence_count) {
-               throw PreprocessingException(
-                  "Sequences in meta data and sequence data for chunk " +
-                  buildChunkName(partition_index, chunk_index) +
-                  " are not equal. The sequence store has " +
-                  std::to_string(sequence_store_sequence_count) + " rows, the metadata store has " +
-                  std::to_string(metadata_store_sequence_count) + " rows."
-               );
+            for (auto& [nuc_name, reference_sequence] : reference_genomes.nucleotide_sequences) {
+               std::filesystem::path sequence_filename = input_folder;
+               sequence_filename += "nuc_" + nuc_name + std::filesystem::path::preferred_separator;
```

Comment:
Do we want to force users to use this abbreviation here? Those files will be generated by scripts mostly anyway, so it would also not hurt much so call the files `nucleotideSequence_main...`.

Reply:
I proposed it as such and Chaoran liked the notation https://github.com/GenSpectrum/LAPIS-SILO/pull/144#issue-1785831275

Reply:
If Chaoran doesn't have a strong opinion on it, I would still vote for the longer name. Our aim is to make this usable for external maintainers, so we should avoid obfuscating things.

Reply:
I think that `nuc` (just like e.g. `aa` or `qc`) is a very common abbreviation for the users/maintainers that we target, so I think that it is fine to use it (but it's not a super strong opinion)

Reply:
Additionally the file ending `fasta` is already the indication that it is a sequence file and we only prepend `gene_` and `nuc_` to avoid collisions between the two.

Reply:
I will add the discussion to #156
Maybe afterall we do make it configurable? An alternative would also be, to let the nucleotideSequence prefix default to the empty string, as the use-case will most often only contain one such sequence. Then the default filename could also be `aligned` to further comply with standard outputs of other software systems